src/utils/setup/aggregate-items-by-video-type.ts

```diff
@@ -0,0 +1,16 @@
+import { Video } from '../../interfaces/video';
+
+export const aggregateItemsByVideoType = (
```

Comment:
Ideally we should use polymorphism here , if not have a separate class/file. I like to avoid if clauses since they always involve us to touch existing prod code.